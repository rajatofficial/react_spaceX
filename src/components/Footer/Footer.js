import React from 'react';
import './Footer.css';

const Footer = () => {
    return(
        <div>
        <h6> Developed By: Rajat Khurana</h6>
        </div>
    )
}

export default Footer;