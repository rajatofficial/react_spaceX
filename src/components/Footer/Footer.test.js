import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer Component', () => {
  it('footer should render correctly in "debug" mode', () => {
    const component = shallow(<Footer debug />);
  
    expect(component).toMatchSnapshot();
  });
});