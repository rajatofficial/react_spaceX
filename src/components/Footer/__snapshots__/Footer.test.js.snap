// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component footer should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
